src/templates/pageComponentsV2/PageBody.jsx

```diff
@@ -1,11 +1,7 @@
-import PropTypes from "prop-types"
-
-export const PageBody = ({ chunk }) => (
-  <div className="col is-8 is-offset-1-desktop is-12-touch print-content page-content-body">
-    <div className="content" dangerouslySetInnerHTML={{ __html: chunk }} />
-  </div>
-)
-
-PageBody.propTypes = {
-  chunk: PropTypes.string,
+export const PageBody = ({ chunk }) => {
```

Comment:
nit: quick win change to tsx?

Reply:
leaving as is. don't want to deal with import updates + not that much incremental value (things already work). 

ideally, we make chunk `TrustedHTML` and this allows us to guarantee that all our HTML is sanitised/safe. otherwise, shifting from `jsx -> tsx` here is abit of a hassle for me